lambdas/db-migration/src/migrations/20200901060127_create_files.ts

```diff
@@ -0,0 +1,23 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => knex.schema
+  .createTable('files', (table) => {
+    table.bigIncrements('cumulusId').comment('Internal Cumulus ID for a file').primary();
+    table.integer('granuleCumulusId').references('cumulusId').inTable('granules').notNullable();
+    table.timestamps(false, true);
+    table.integer('fileSize').comment('Deprecated - size of file');
+    table.integer('size').comment('Size of file (bytes)');
+    table.text('bucket').comment('AWS Bucket file is archived in');
+    table.text('checksumType').comment('Type of file checksum (e.g. md5');
```

Comment:
I'm fine with the comments, but it might be a little more readable if it were split across lines:

```suggestion
    table
      .text('checksumType')
      .comment('Type of file checksum (e.g. md5');
```

Reply:
Will do across all files